src/util/namespace.h

```diff
@@ -84,8 +84,14 @@ class namespacet:public namespace_baset
 
   using namespace_baset::lookup;
 
-  // these do the actual lookup
+
+  /// Searches for a symbol named \p name in `symbol_table1` and returns a
+  /// pointer to it in output parameter \p symbol when found. When no symbol is
+  /// found, it performs the same lookup in `symbol_table2`.
+  /// \return False iff the requested symbols is found in one of the two tables.
   virtual bool lookup(const irep_idt &name, const symbolt *&symbol) const;
+
+  // this and lookup above do the actual lookup
```

Comment:
This was an old comment talking about the method below (undocumented) and the method above (which I documented). Since I don't know enough about the second method I decided to leave the comment for someone else to address it.

If you can produce a description of the method below I can add it here.